vsphere-cpi.html.md.erb

```diff
@@ -58,7 +58,7 @@ networks:
 vSphere CPI does not support dynamic or vip networks.
 
 ---
-## <a id='resource-pools'></a> Resource Pools / VM Types
+## <a id='resource-pools'></a>VM Types (Resource Pools)
```

Comment:
We reverted the Resource Pool to VM Type change to more consistent with the rest of the docs.